docs/notes/vineyard-operator.rst

```diff
@@ -1207,43 +1224,41 @@ daemon server with spill mechanism:
     kind: Vineyardd
     metadata:
       name: vineyardd-sample
+      # don't use default namespace
       namespace: vineyard-system
     spec:
-      image: ghcr.io/v6d-io/v6d/vineyardd:alpine-latest
-      replicas: 3
-      imagePullPolicy: IfNotPresent
-      # vineyardd's configuration
-      sharedMemorySize: "2048M"
-      syncCRDs: true
-      enableMetrics: false
-      # spill configuration
-      spillConfig:
-        # if set, then enable the spill mechanism
-        name: spill-path
-        # please make sure the path exists
-        path: /var/vineyard/spill
-        spillLowerRate: "0.3"
-        spillUpperRate: "0.8"
-        persistentVolumeSpec:
-          storageClassName: manual
-          capacity:
-            storage: 1Gi
-          accessModes:
-            - ReadWriteOnce
-          hostPath:
-            path: /var/vineyard/spill
-      persistentVolumeClaimSpec:
-          storageClassName: manual
-          accessModes:
-            - ReadWriteOnce
-          resources:
-            requests:
-              storage: 512Mi
```

Comment:
I remembered that using a flatten configuure structure is by-design. Have we rollbacked the change and use `vineyardConfig` entry again?

Reply:
Yes, we use the `vineyardConfig ` as a common struct shared by CRD `Vineyardd` and `Sidecar`.